src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -263,63 +263,68 @@
 					</AnimatedIcon>
 				</ToggleButton>
 
-				<Button
-					x:Name="Back"
```

Comment:
Why is this changed?

Reply:
To correct arrow direction in RTL
![image](https://github.com/user-attachments/assets/6edf6c2b-9466-4b7e-b107-410095c68015)

Reply:
@yaira2 fyi

Reply:
What happens without this change?

Reply:
The arrows are facing each other

Reply:
Would it work to wrap these in a StackPanel (as you've done) and hardcode the FlowDirection of the StackPanel?

Reply:
Yes this is exactly the solution: `FlowDirection="{Binding Path=FlowDirection, ElementName=Root}"` that I use